package.json

```diff
@@ -36,38 +36,38 @@
     "focus-trap-react": "^3.1.4",
     "glamor": "^2.20.40",
     "keycode": "^2.1.8",
-    "lodash": "^4.17.15",
-    "moment": "^2.26.0",
-    "moment-timezone": "^0.5.31",
```

Comment:
We upgraded lodash to 4.17.19, jest to 26.1.0, which requires a higher version of node, which we should be on anyway at that this point (I updated the travis file). We also ran npm audit fix which fixed some nested deps.